include/cuda/std/detail/libcxx/include/optional

```diff
@@ -317,24 +374,28 @@ struct __optional_storage_base : __optional_destruct_base<_Tp>
 
     template <class... _Args>
     _LIBCUDACXX_INLINE_VISIBILITY
-    void __construct(_Args&&... __args)
+    _LIBCUDACXX_CONSTEXPR_AFTER_CXX11 void __construct(_Args&&... __args)
```

Comment:
Ditto, this should be 
```suggestion
    _LIBCUDACXX_CONSTEXPR_AFTER_CXX17 void __construct(_Args&&... __args)
```

Reply:
Ditto.

Reply:
This is calling placement new, that cannot be constexpr or am I missing something?